### PR TITLE
House keeping 08 11 2021

### PR DESCRIPTION
**Pull request by Mercurial**
- Updated to docusaurus beta.4
- Attemp to fix dependecy security alerts
- Updated correct cardano-node / cli binaries and configuration links

### CODE REVIEW

File: src/data/builder-tools.js [hunk 1]
```diff
@@ -161,6 +161,12 @@ const Showcases = [
     website: "https://cardano-community.github.io/guild-operators/",
     getstarted: "/docs/operate-a-stake-pool/guild-ops-suite",
     tags: ["featured", "getstarted", "operatortool"],
+    title: "CNTools",
```
Comment (rdlrt) [inline src/data/builder-tools.js:164]: @Mercurial - You might have accidentally used previous commit as base

**Review by katomm — APPROVED** [review]

File: src/data/builder-tools.js (hunk 1 above)
Comment (Mercurial) [reply src/data/builder-tools.js:164]: oh I merged it wrong should be a quick fix

**Review by katomm — APPROVED** [review]

**Review by rdlrt — APPROVED** [review]
LGTM

**Review by katomm — APPROVED** [review]

**Review by gufmar — APPROVED** [review]
Lgtm

**Pull request merged**